src/ChipInput.js

```diff
@@ -423,6 +479,7 @@ class ChipInput extends React.Component {
       helperText,
       id,
       InputProps = {},
+      inputAdornmentRender = defaultInputAdornmentRenderer,
```

Comment:
This should be called render*er*

```suggestion
      inputAdornmentRenderer = defaultInputAdornmentRenderer,
```